CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Here is a template for new release sections
 - Function `server.run_sensitivity_analysis_step` to perform one step of a sensitivity analysis (#936)
 - Function `utils.nested_dict_crawler` to return mapping of path within a nested dict to the keys at the lowest nested level (#936)
 - Test `test_utils.TestAccessKPIs` to test the nested dict utils functions (#936)
+- `sankey` method to the `ESGraphRenderer` class to return a sankey diagram (#935)
+- `plot_sankey_diagramm` function in `D0_modelling_and_optimization.py` to add the dict of the sankey diagram
 
```

Comment:
- `ready_sankey_diagram` in `F2_autoreport.py` to add sankey diagram to output and report (#935)